src/matrixHelpers/FormioLoader/index.js

```diff
@@ -0,0 +1,202 @@
+  } else {
+    formName = opts.formName;
+  }
+  const { projectName } = opts;
+  const formConfirmation = opts.formConfirmation || defaultRedirect;
+  const namespace = opts.namespace || `formio-${projectName}`;
+  const formUrl = `${baseUrl}/${projectName}/${formName}`;
+
+  /*
+   * init formio instance
+   */
+  const formio = new Formio(formUrl, {
+    base: baseUrl,
+    project: `${baseUrl}/${projectName}`,
+    namespace,
+  });
+  formioElem.dataset.formio = JSON.stringify(formio);
+  formioElem.dataset.formioFormUrl = formUrl;
+
+  /*
+   * load formio form
+   */
+  Formio.createForm(
+    formioElem,
+    formUrl,
+    // form,
+    {
+      // Turn off default buttons
+      buttonSettings: {
+        showCancel: false,
+        showPrevious: false,
+        showNext: false,
+        showSubmit: false,
+      },
+      i18n: {
+        en: { pattern: "Must use the format shown" },
+      },
+      formio,
+      namespace: formio.options.namespace,
+    }
+  ).then((wizard) => {
+    wizard.formio = formio;
+    wizard.options.formio = formio;
+
+    // eslint-disable-next-line no-underscore-dangle
+    const formTitle = wizard._form.title;
+    // eslint-disable-next-line no-underscore-dangle
+    const formModified = wizard._form.modified;
+
+    // Force new tab on formlinks
+    bodyContainer.on("click", `#${formioContainerId} a`, (e) => {
+      e.target.target = "_blank";
+    });
+
+    // Change event/GTM
+    wizard.on("click", (change) => {
+      const changeObj = change;
+      if (
+        typeof changeObj.changed !== "undefined" &&
+        typeof changeObj.changed.component !== "undefined"
+      ) {
+        window.dataLayer.push({
+          event: "formio-interaction",
+          "formio-name": formTitle,
+          "formio-input-id": changeObj.changed.component.id,
+          "formio-input-type": changeObj.changed.component.type,
+          "formio-input-value": changeObj.changed.component.value,
+          "formio-input-key": changeObj.changed.component.key,
+          "formio-input-label-raw": changeObj.changed.component.label,
+          "formio-version": formModified,
+          "formio-category": `Form: ${formTitle}`,
+          "formio-action": "Value changed",
+        });
+      }
+    });
+
+    // Must use 'applicationSubmit' custom event on primary submit
+    wizard.on("applicationSubmit", () => {
+      submitBtn.attr("disabled", true);
+      wizard
+        .submit()
+        .then(() => {
+          if (formConfirmation) {
+            window.location.href = `/${formConfirmation}`;
+          } else {
+            // No confirmation set. Using generic redirection
+            window.location.href = defaultRedirect;
+          }
+        })
+        .catch(() => {
+          console.debug("Submission error");
+        });
+    });
+  });
+};
+
+// polyfill plugin function to fix the namespace option doesn't pass to Formio.makeRequest
+const NamespacePolyfillPlugin = {
```

Comment:
IE11 is now not a requirement since mid march this year. so your good to go on ripping out the polyfill bits in another pull request.

Reply:
the name may be confusing... this `polyfill` is for filling the gap of the namspace inplementation in Formio instance, i better rename it to just `NamespacePlugin `